src/views/Page404/components/ScottieDog/scottieDog.scss

```diff
@@ -0,0 +1,114 @@
+$dog-color: #555;
+$dog-body-width: 10em;
+$dog-body-height: 5em;
+
+.dog {
+    position: relative;
+    /*** font size of root element controls size of dog ***/
+    // font-size: 30px;
+
+    /*** Used for walking across screen animation (optional) ***/
+    animation: dog-walk linear 30s;
+    animation-delay: 5s;
+    left: 150%;
+}
+@keyframes dog-walk {
+    100% {
+        left: -50%;
+    }
+}
+.dog * {
+    position: absolute;
+}
+.dog-body {
+    background-color: $dog-color;
+    width: $dog-body-width;
+    height: $dog-body-height;
+    top: 0;
+    border-radius: 0 1.5em 0 1.5em;
+}
+.dog-head {
+    background-color: $dog-color;
+    height: calc(#{$dog-body-height} / 1.6);
+    width: calc(#{$dog-body-width} / 1.8);
+    left: 0;
+    top: 0;
+    transform: translateX(calc(#{-1 * $dog-body-width} / 3.2)) translateY(calc(#{-1* $dog-body-height} / 2.5)) rotate(-15deg);
+    border-radius: 0.2em;
+}
+.dog-eye {
+    background-color: #ddd;
+    width: 0.7em;
+    height: 0.7em;
+    border-radius: 50%;
+    right: 30%;
+    top: 15%;
+}
+.dog-nose {
+    background-color: #111;
+    width: 0.9em;
+    height: 0.9em;
+    border-radius: 50%;
+    transform: translateX(-0.2em) translateY(-0.2em);
+}
+.dog-ear {
+    right: 0;
+    top: 0;
+    transform: translateY(-1.5em) rotate(15deg);
+    width: 0;
+    height: 0;
+    border-left: solid 1em transparent;
+    border-right: solid 1em transparent;
+    border-bottom: solid 2em $dog-color;
+}
+.dog-collar {
+    background-color: rgb(165, 0, 0);
+    bottom: 0;
+    right: 0;
+    width: 50%;
+    height: 2.5em;
+    transform: rotate(-8deg);
+    font-size: 0.3em;
+    line-height: 2.5em;
+    color: #fff;
+    text-align: center;
+    font-family: monospace;
+}
+.dog-leg {
+    background-color: $dog-color;
+    width: 1.5em;
+    height: 1.5em;
+    top: 5em;
+    left: 0;
+}
+.dog-leg1 {
+    transform: translateX(2em);
+}
+.dog-leg2 {
+    transform: translateX(4em);
+    height: 1.2em;
+    background-color: #444;
+}
+.dog-leg3 {
+    transform: translateX($dog-body-width - 3.5em);
+}
+.dog-leg4 {
+    transform: translateX($dog-body-width - 1.5em);
+    height: 1.2em;
+    background-color: #444;
+}
+.dog-tail {
+    transform: translateX($dog-body-width - 2em + 0.8em) translateY(-4.5em + 1em) rotate(30deg);
+    width: 0;
+    height: 0;
+    border-left: solid 0.5em transparent;
+    border-right: solid 0.5em transparent;
+    border-bottom: solid 4.5em $dog-color;
+    transform-origin: bottom;
+    animation: wag-tail 1s infinite;
+}
+@keyframes wag-tail {
+    50% {
+        transform: translateX($dog-body-width - 2em + 0.8em) translateY(-4.5em + 1em) rotate(10deg);
+    }
+}
```

Comment:
.scss to .css should be automatic in the build process, and it seems to be working here.

Finishing the file with a newline is nice because it's logical and consistent, and because some tools (more often on Linux than Windows) care.  I don't think the absence breaks anything in our typical work flow.